File: app/src/main/java/com/keplersegg/myself/Async/ISignOut.java
package com.keplersegg.myself.Async;

import com.google.android.gms.common.api.GoogleApiClient;
import com.keplersegg.myself.Models.User;

public interface ISignOut {

    void onSignOut();

    User GetUser();

    GoogleApiClient GetGoogleApiClient();
}

File: app/src/main/java/com/keplersegg/myself/Async/ISignOut.kt
package com.keplersegg.myself.Async

import com.google.android.gms.common.api.GoogleApiClient
import com.keplersegg.myself.Models.User

interface ISignOut {

    fun onSignOut()

    fun GetUser(): User?

    fun GetGoogleApiClient(): GoogleApiClient
}
